formbricks/formbricks
# fix: fixes survey link share modal and mobile nav menu

## What does this PR do?



Fixes #617 (issue)

https://github.com/formbricks/formbricks/assets/54475686/d7382e31-8c91-4cbc-9818-a19bd7fef43b

## Type of change




## How should this be tested?



- Make a link survey or go to the survey page of an existing link survey
- Click the Share button and the share modal should open on mobile and work correctly
- Navigation Menu on mobile should contain the formbricks logo and a dropdown menu for navigation

## Checklist
